## Description

leej3 opened the pull request:
<!-- If this is a bug-fix or enhancement, state the issue # it closes -->
<!-- If this is a new feature, reference what paper it implements. -->


## Checklist

<!-- It's fine to submit PRs which are a work in progress! -->
<!-- But before they are merged, all PRs should provide: -->
- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

<!-- For detailed information on these and other aspects see -->
<!-- the scikit-image contribution guidelines. -->
<!-- https://scikit-image.org/docs/dev/contribute.html -->

## For reviewers

<!-- Don't remove the checklist below. -->
- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

leej3 commented on RELEASE.txt:
```diff
@@ -71,22 +67,14 @@ access), specifically to:
 
- - On the main branch, edit ``doc/source/_static/docversions.js``,
+- On the main branch, edit ``doc/source/_static/docversions.js``,
   add the release, e.g., `0.15.x`, and commit.  Cherry-pick this
```
does docversions.js still need to be modified for now?

viniciusdc replied:
We haven't worked with anything doc related so far...

leej3 replied:
Ah yes. Noted.

leej3 commented on RELEASE.txt:
```diff
@@ -71,22 +67,14 @@ access), specifically to:
   are building fine. Then proceed with the release.
 
-    1. Back on the release branch, update the version number to stable in
+    1. Update the version number to stable in
```
is this automated now? can it be automated?

viniciusdc replied:
Yes it can be automated, but I would prefer to have that as part of the release process as cibuildwheels uses that version string to generate the wheels files

leej3 replied:
Ok a pr once towncrier is done would be good.

sciunto committed:
Fix numpy deprecation in an example (#5429)

* fix numpy deprecation

viniciusdc commented on RELEASE.txt:
```diff
@@ -103,42 +91,8 @@ access), specifically to:
-  ``scikit_image-<versiontag>-<platforminfo>.whl``
-
-**Do not tag until the builds are all green.**
+    4. Iterate until the CI building and testing of the wheels is to your
```
Actually if you want to test the wheels prior to release (as a release candidate) you will need to push the tag cibuildwheels to only trigger the build wheels part. If you push the version tag as vx.y.z. it will trigger the release process.

leej3 replied:
Good point thanks.

viniciusdc commented on RELEASE.txt:
```diff
@@ -149,41 +103,17 @@ access), specifically to:
-  Choose the tag corresponding to this release, copy the release notes,
-  and upload the wheels in the appropriate slot. Finally, click on
-  "Publish release".
+- Publish the release on github
```
As a future addition it will generate the release notes for inspection too, by using TownCrier

viniciusdc commented on RELEASE.txt:
```diff
@@ -149,41 +103,17 @@ access), specifically to:
+- Publish the release on github
 
 - Update the release docs:
    - Edit ``doc/source/_static/docversions.js`` and commit
```
We can trigger the auto docs once the auto release CI finishes

leej3 committed:
Make buildwheel a tag pattern in CI workflow

trigger buildwheel workflow off any tag that starts with buildwheels

leej3 committed:
update RELEASE.txt

This update to the docs reflects the changes to the release process
implemented in https://github.com/scikit-image/scikit-image/pull/5397

Commit:
Update RELEASE.txt

Co-authored-by: Marianne Corvellec <marianne.corvellec@ens-lyon.org>

Commit:
Update RELEASE.txt

Co-authored-by: Gregory R. Lee <grlee77@gmail.com>
